spread/project.go

```diff
@@ -124,6 +124,7 @@ type System struct {
 
 	// Only for Google so far.
 	SecureBoot bool `yaml:"secure-boot"`
+	Plan       string
```

Comment:
It's not true that only Google has this, I believe.

Reply:
Otherwise, it looks reasonable. A unit test about the new and old behavior would also be welcome.

Reply:
I added a unit test now but it does not fully cover old/new because for that it would need to test createMachine() which involves a bunch of mocking. I can do this but I am a bit worried it might be a bit much for a this (simple) PR.